src/direct.rs

```diff
@@ -25,7 +25,9 @@ mod tests {
     use std::collections::HashMap;
 
     use ruma_identifiers::{RoomId, UserId};
-    use serde_json::{from_value, json, to_value};
+    use serde_json::from_value as from_json_value;
+    use serde_json::json;
+    use serde_json::to_value as to_json_value;
```

Comment:
You don't need to split up the import to rename items in it.

```suggestion
    use serde_json::{from_value as from_json_value, json, to_value as to_json_value};
```

Reply:
oh i see, ill fix that up real quick